### PR TITLE
consolidate firmware folder in smoll parent folder

### PR DESCRIPTION
**Pull request by SmollChungus**
Consolidate firmware folders into the smoll parent folder, in preparation for future additions.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

**Comment by dunk2k** [comment]
@SmollChungus 
Could respective entries, for moved keyboards, be added in `data/mappings/keyboard_aliases.hjson`?

**Comment by SmollChungus** [comment]
> @SmollChungus Could respective entries, for moved keyboards, be added in `data/mappings/keyboard_aliases.hjson`?

Yes for sure! Is this something I would need to add to this PR / open another PR for?

**Comment by dunk2k** [comment]
> > @SmollChungus Could respective entries, for moved keyboards, be added in `data/mappings/keyboard_aliases.hjson`?
> 
> Yes for sure! Is this something I would need to add to this PR / open another PR for?

With this PR please 😊

**Comment by SmollChungus** [comment]
Okay perfect, added modified mappings to the PR!

**Review by drashna — APPROVED** [review]

File: keyboards/smoll/lefty/rules.mk
Comment (zvecr) [inline keyboards/smoll/lefty/rules.mk:1]: `DEFAULT_FOLDER = lefty/rev2` in this file is incorrect and needs to be updated to `smoll/lefty/rev2`.

**Review by zvecr — CHANGES_REQUESTED** [review]

File: data/mappings/keyboard_aliases.hjson [hunk 1]
```diff
@@ -924,6 +924,9 @@
     "launchpad/rev1": {
         "target": "maple_computing/launchpad/rev1"
     },
+    "lefty": {
+        "target": "smoll/lefty/rev2"
+    },
```
Comment (dunk2k) [inline data/mappings/keyboard_aliases.hjson:929]:
```suggestion
    },
    "lefty/rev1": {
        "target": "smoll/lefty/rev1"
    },
```

**Review by dunk2k — CHANGES_REQUESTED** [review]

**Review by zvecr — APPROVED** [review]

**Pull request merged**